<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterForm</class>
 <widget class="QDialog" name="filterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="appLabel">
       <property name="text">
        <string>Application:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="appEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="processIdLabel">
       <property name="text">
        <string>Process-Id:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="threadIdLabel">
       <property name="text">
        <string>Thread-Id:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="functionLabel">
       <property name="text">
        <string>Class/Function:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="messageLabel">
       <property name="text">
        <string>Message:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="typeCombo"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pidEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="tidEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="funcEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="messageEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>appEdit</tabstop>
  <tabstop>pidEdit</tabstop>
  <tabstop>tidEdit</tabstop>
  <tabstop>funcEdit</tabstop>
  <tabstop>messageEdit</tabstop>
  <tabstop>typeCombo</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>filterForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>filterForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
